packages/manager/modules/preloader/rollup.config.js

```diff
@@ -0,0 +1,28 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const outputs = [
+  config.es({
+    output: {
+      sourcemap: false,
+    },
+  }),
+];
+
+if (process.env.BUILD === 'production') {
+  outputs.push(config.cjs());
+  outputs.push(
+    config.umd({
+      output: {
+        globals: {
+          angular: 'angular',
```

Comment:
We can get rid of this globals object I think.

Reply:
removed.